### ISSUE TITLE
Add node_modules folder to .gitignore for new Node.js scrapers

### ISSUE THREAD

**Issue by drzax** [openaustralia/morph#889]
This folder should almost never be under version control. Node's dependency management is very good and there's rarely a need to commit dependencies. In fact, it often causes problems since many dependencies are compiled specifically for the platform.

**Comment by henare** [comment]
Good call. In fact we should probably do this for all languages: https://github.com/github/gitignore

**Comment by equivalentideas** [comment]
If you have time, it would be great if you could submit a PR for this @drzax ;-)

**Comment by drzax** [comment]
I had a look around in the code, but there doesn't seem to be any language specific template for the .gitignore file.





I'll see if I can find some time soon.
